odoo/addons/test_new_api/tests/test_new_fields.py

```diff
@@ -576,16 +575,6 @@ def test_13_inverse_access(self):
         with self.assertRaises(AccessError):
             foo.with_user(user).display_name = 'Forbidden'
 
-    def test_13_inverse_access(self):
```

Comment:
think we fixed this duplicate in 14.0 when I introduced some pylint improvements, check the fw-port for conflicts :D